applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -0,0 +1,1596 @@
+* `TCP server #XTCPSVR`_
+* `TCP/TLS client #XTCPCLI`_
+* `TCP send data #XTCPSEND`_
+* `TCP receive data #XTCPRECV`_
+* `UDP server #XUDPSVR`_
+* `UDP/DTLS client #XUDPCLI`_
+* `UDP send data #XUDPSEND`_
+
+For more information on the BSD networking services, visit the `BSD Networking Services Spec Reference`_.
+
+BSD socket #XSOCKET
+===================
+
+The ``#XSOCKET`` command allows you to open or close a socket, or to check the socket handle.
+
+Set command
+-----------
+
+The set command allows you to open or close a socket.
+
+Syntax
+~~~~~~
+
+::
+
+   #XSOCKET=<op>[,<type>,<role>[,<sec_tag>]]
+
+* The ``<op>`` parameter can accept one of the following values:
+
+  * ``0`` - Open
+  * ``1`` - Close
+
+* The ``<sec_tag>`` parameter is an integer.
+  It indicates to the modem the credential of the security tag used for establishing a secure connection.
+  It is associated with the certificate or PSK.
+  Specifying the ``<sec_tag>`` is mandatory when opening a socket.
+  When TLS/DTLS is expected, the credentials should be stored on the modem side by ``AT%XCMNG`` or by the Nordic nRF Connect/LTE Link Monitor tool.
+  The modem needs to be in the offline state.
+  The DTLS server is not supported.
+
+* The ``<type>`` parameter value depends on the presence of the <sec_tag> parameter.
+  When the ``<sec_tag>`` is not specified:
+
+  * 1: SOCK_STREAM for TCP
+  * 2: SOCK_DGRAM for UDP
+
+  When the ``<sec_tag>`` is specified:
+
+  * 1: SOCK_STREAM for TLS
+  * 2: SOCK_DGRAM for DTLS
+
+* The ``<role>`` parameter can accept one of the following values:
+
+  * ``0`` - Client
+  * ``1`` - Server
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XSOCKET: <handle>[,<type>,<protocol>]
+
+* The ``<handle>`` value is an integer.
+  It can be interpreted as follows:
+
+  * Positive - The socket opened successfully.
+  * Negative - The socket failed to open.
+  * ``0`` - The socket closed successfully.
+
+* The ``<type>`` parameter value depends on the presence of the <sec_tag> parameter.
+  When the ``<sec_tag>`` is not specified:
+
+  * 1: SOCK_STREAM for TCP
+  * 2: SOCK_DGRAM for UDP
+
+  When the ``<sec_tag>`` is specified:
+
+  * 1: SOCK_STREAM for TLS
+  * 2: SOCK_DGRAM for DTLS
+
+* The ``<protocol>`` value is present only in the response to a request to open the socket.
+  It can be one of the following:
+
+  * ``6`` - IPPROTO_TCP
+  * ``17`` - IPPROTO_UDP
+  * ``258`` - IPPROTO_TLS_1_2
+  * ``273`` - IPPROTO_DTLS_1_2
+
+Unsolicited notification
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+   #XSOCKET: <error> closed
+
+The ``<error>`` value is a negative integer.
+It represents the error value according to the standard POSIX *errorno*.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XSOCKET=1,1,0
+   #XSOCKET: 3, 6, 0
+   OK
+   AT#XSOCKET=1,2,0
+   #XSOCKET: 3, 17, 0
+   OK
+   AT#XSOCKET=0
+   #XSOCKET: 0, closed
+   OK
+   at#xsocket=1,1,0,16842755
+   #XSOCKET: 2, 1, 0, 258
+   OK
+   at#xsocket=1,2,0,16842755
+   #XSOCKET: 2, 2, 0, 273
```

Comment:
I suggest to replace 16842755 by 16842753. 16842753 is default sec_tag value of nRF cloud.

http://developer.nordicsemi.com/.NCS_PV/doc/nrf/reference/kconfig/CONFIG_NRF_CLOUD_SEC_TAG.html